src/test/java/teammates/test/cases/ui/browsertests/InstructorCommentsPageUiTest.java

```diff
@@ -179,6 +180,7 @@ private void testPanelsCollapseExpand() {
     private void testActions() throws Exception {
         ______TS("action: edit student comment");
         commentsPage.clickStudentCommentEditForRow(1);
+        ThreadHelper.waitFor(1000);
```

Comment:
It waits for TinyMCE initialization, because lazy initialization is used i.e. only after showing edit row